.circleci/ci_secrets.yml

```
@@ -1,6 +1,6 @@
 db:
   name: circleci_test
   user: circleci
-  password: ''
+  password: password
```

I think we have to change it here too:

```suggestion
  password: not_secure
```